src/pages/iou/steps/NewRequestAmountPage.tsx

```diff
@@ -88,29 +74,29 @@ function NewRequestAmountPage({route, iou, report, selectedTab}) {
     useEffect(() => {
         if (isEditing) {
             // ID in Onyx could change by initiating a new request in a separate browser tab or completing a request
-            if (prevMoneyRequestID.current !== iou.id) {
+            if (prevMoneyRequestID.current !== iou?.id) {
                 // The ID is cleared on completing a request. In that case, we will do nothing.
-                if (!iou.id) {
+                if (!iou?.id) {
                     return;
                 }
-                Navigation.goBack(ROUTES.MONEY_REQUEST.getRoute(iouType, reportID), true);
+                Navigation.goBack(ROUTES.MONEY_REQUEST_CONFIRMATION.getRoute(iouType, report?.reportID), true);
```

Comment:
Why this change? MONEY_REQUEST -> MONEY_REQUEST_CONFIRMATION

Reply:
@blazejkustra @fabioh8010 `ROUTES.MONEY_REQUEST` was removed and according to the logic, it should be replaced with `ROUTES.MONEY_REQUEST_CONFIRMATION`, but honestly I think this page is going to be removed in the future because I don't see it being used.